app/views/layouts/home.html.erb

```diff
@@ -4,16 +4,22 @@
     <%= render 'shared/head' %>
   </head>
   <body>
-    <div class='fixed-top'>
-      <%= render 'shared/nav' %>
-    </div>
+    <% if Flipper.enabled?(:navigation, current_user, current_user.school_group) %>
+      <%= render 'shared/navigation/top' %>
+    <% else %>
+      <div class='fixed-top'>
+        <%= render 'shared/nav' %>
+      </div>
+    <% end %>
     <% if flash.any? %>
       <section class="home alerts">
         <%= render 'shared/alerts' %>
       </section>
     <% end %>
     <!-- don't render application container as you do for the other pages -->
-    <%= yield %>
+    <div class="<%= nav_margin_classes %>">
```

Comment:
Put this here rather than amend every file that uses the home layout (there are quite a few!)